### PR TITLE
Remove unnecessary <sstream> header include.

### PR DESCRIPTION
**Pull request by bangerth**
None of these tests using `std::[io]stringstream`.

### CODE REVIEW

File: tests/mpi/multigrid_adaptive.cc [hunk 1]
```diff
@@ -62,7 +62,6 @@
 #include "../tests.h"
 
 // This is C++:
```
Comment (marcfehling) [inline tests/mpi/multigrid_adaptive.cc:64]:
```suggestion
```

File: tests/mpi/multigrid_uniform.cc [hunk 2]
```diff
@@ -62,7 +62,6 @@
 #include "../tests.h"
 
 // This is C++:
```
Comment (marcfehling) [inline tests/mpi/multigrid_uniform.cc:64]:
```suggestion
```

**Review by marcfehling — APPROVED** [review]
Looks good after you remove the two outdated comments

**Comment by bangerth** [comment]
So done!

**Review by marcfehling — APPROVED** [review]

**Pull request merged**